dask/dataframe/dispatch.py

```diff
@@ -8,6 +8,7 @@
 
 import dask.array as da
 import dask.dataframe as dd
+from dask.array.dispatch import percentile_dispatch  # noqa: F401
```

Comment:
I think this was recently added and I suspect is only being used in RAPIDS. Can we remove this import and make a corresponding update in RAPIDS?